internal/indexmanagement/reconcile.go

```diff
@@ -362,6 +362,7 @@ func (imr *IndexManagementRequest) reconcileIndexManagementCronjob(policy apis.I
 		envvars = append(envvars,
 			corev1.EnvVar{Name: "MIN_AGE", Value: strconv.FormatUint(minAgeMillis, 10)},
 			corev1.EnvVar{Name: "NAMESPACE_SPECS", Value: namespaceSpecsString},
+			corev1.EnvVar{Name: "DISK_THRESHOLD", Value: strconv.FormatInt(policy.Phases.Delete.DiskThresholdPercent, 10)},
```

Comment:
Can we please do some validation here on `policy.Phases.Delete.DiskThresholdPercent`. For example, the % value should be between 0-100 otherwise return an error.